dashboard/db/migrate/20180129220540_rename_create_at_to_featured_at.rb

```diff
@@ -0,0 +1,5 @@
+class RenameCreateAtToFeaturedAt < ActiveRecord::Migration[5.0]
+  def change
+    rename_column :featured_projects, :created_at, :featured_at
+  end
```

Comment:
I didn't notice this on the first pass. Is there a reason we want to rename this away from the Rails convention?

Rails has a [timestamp convention](http://api.rubyonrails.org/classes/ActiveRecord/Timestamp.html) that `created_at` and `updated_at` columns are automatically updated. In our case we don't need `updated_at` since we only care about a specific type of update (unfeatured) which we will need to set explicitly, but we could still leverage `created_at` here. If we rename this field, in addition to it being less standard we'll need to add code to set it on create. 

I'd suggest leaving this as `created_at` and if we want a `featured_at` method, just create an [alias](https://apidock.com/rails/Module/alias_attribute).